### PR TITLE
EOS D60 noise profile

### LINKED ISSUE #9300: Can't saturated blue channel for noise profile - Canon EOS D60
In the link I have some couple of raw files from Canon EOS D60. It seems that no matter what I do I will not be able to saturate the blue channel at ISO 200 and ISO 400.
Skärmbild från 2021-06-21 12-58-48

CC0 https://drive.google.com/file/d/1TTQhR6Q18FwRm21Jqbbw6NRKuCVQCNYn/view?usp=sharing

### PR DESCRIPTION
**Pull request by rawfiner**
Fixes #9300

### CODE REVIEW

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -1201,6 +1201,17 @@
...
         },
+        {
+          "comment": "eos d60 contributed by PeterWem",
+          "model": "EOS D60",
+          "profiles": [
+            {"name": "EOS D60 iso 100", "iso": 100, "a": [1.36406478037885e-05, 4.71801288650747e-06, 4.49147092998427e-06], "b": [-1.79594608254617e-09, 4.93714256557499e-09, 3.77423652251075e-09]},
+            {"name": "EOS D60 iso 200", "iso": 200, "a": [1.64922309690471e-05, 5.93512833354441e-06, 8.71280849716513e-06], "b": [1.63736758264013e-10, 5.43488061070343e-10, 6.10413895337077e-10]},
+            {"name": "EOS D60 iso 400", "iso": 400, "a": [3.58147073617352e-05, 1.19811066798068e-05, 1.58984730041545e-05], "b": [4.61416104976025e-09, 6.17963414176452e-09, 1.31227683636844e-08]},
+            {"name": "EOS D60 iso 800", "iso": 800, "a": [6.79834799910035e-05, 2.21390988276476e-05, 3.37964490618815e-05], "b": [8.1311148938383e-09, 1.65102862462688e-08, 1.42532815290596e-08]},
+            {"name": "EOS D60 iso 1007", "iso": 1007, "a": [6.70409177756273e-05, 2.13249028535912e-05, 2.80806894890406e-05], "b": [4.95818689559605e-09, 1.39978489467886e-08, 1.19469599534896e-08]}
```
Comment (TurboGit) [inline data/noiseprofiles.json:1212]: ISO **1007**? Is that a typo?

Comment (rawfiner) [reply data/noiseprofiles.json:1212]: No, I was surprised too but that's the value reported in the exifs

Comment (TurboGit) [reply data/noiseprofiles.json:1212]: Ok, but I bet for an exif or in camera software bug. Anyway as this camera is old we'll have to live with that.

**Review by TurboGit — APPROVED** [review]
Thanks!

**Pull request merged**